.github/workflows/ci.yml

```diff
@@ -79,6 +79,44 @@ jobs:
         run: |
           ./vendor/bin/codacycoverage clover build/coverage/xml
 
+  phpcs:
+    name: PHP_CodeSniffer
+
+    runs-on: ubuntu-20.04
+
+    needs: [ php-lint ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: "composer:v2, phive"
+          coverage: none
+
+      - name: Cache dependencies installed with composer
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/composer
+          key: php${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}
```

Comment:
I don’t know too much about github actions still but it seems counter-intuitive to me that `matrix.php-version` would be defined in this action that doesn’t even use matrix.

Reply:
Oh, that's a copy'n'paste error. Good catch!

Reply:
I have now moved the PHP version to a (single-element) matrix so we have exactly one place for the PHP version (which is used both in the PHP install step as well as the caching step) and so that it's consistent with the structure of the other jobs.